modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -220,7 +220,9 @@ get_exec_args() {
 # Ensure that the running UID has the "jboss" passwd metadata
 # XXX: Maybe we should make this an entrypoint for the image?
 function configure_passwd() {
-  sed "/^jboss/s/[^:]*/$(id -u)/3" /etc/passwd > "$HOME/passwd"
+  if [ -w "$HOME/passwd" ]; then
```

Comment:
Please add a comment as to why that check is needed. It's easy to forget why it was added otherwise.

Reply:
Good idea.